src/textual/widgets/_input.py

```diff
@@ -113,6 +113,17 @@ def __init__(
         id: str | None = None,
         classes: str | None = None,
     ) -> None:
+        """Initialise the `Input` widget.
```

Comment:
Does this not need to go in the class docstrings to be picked up?

Reply:
I don't believe so; although it is an alternative approach that's permitted by the Google standard IIRC.

![Screenshot 2023-01-16 at 16 28 01](https://user-images.githubusercontent.com/28237/212725884-215dd313-d895-4d76-96a7-a8090bd0daca.png)

Reply:
Great. I think the Rich docs require the params on the class. But this is better.